INDI/INDI/indiserver.c

```diff
@@ -1040,12 +1037,7 @@ static void indiFIFO(void)
 int isDeviceInDriver(const char *dev, DvrInfo *dp)
 {
     int i = 0;
-    for (i = 0; i < dp->ndev; i++)
-    {
-        if (!strcmp(dev, dp->dev[i]))
-            return 1;
-    }
-
+    for (i=0; i < dp->ndev && strcmp(dev,dp->dev[i]); ++i) { return 1; }
```

Comment:
I am not sure how or why this code works, but functionally this implementation change should not change the behavior.